### DISCUSSION TITLE
本地部署

### DISCUSSION THREAD

**Discussion by zyklone4096** [bs-community/blessing-skin-server#513] [Q&A]
不确定是不是bug

用的phpstudy，MySQL8.0.12和Apache2.4.39

安装报错SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 1000 bytes (SQL: alter table `notifications` add index `notifications_notifiable_type_notifiable_id_index`(`notifiable_type`, `notifiable_id`))

**Comment by Big-Cake-jpg** [comment]
请客官看文档。
